setup.cfg

```
@@ -42,7 +42,7 @@ packages = find:
 install_requires =
     backports.entry-points-selectable>=1.0.4
     distlib>=0.3.1,<1
-    filelock>=3.2,<4
+    filelock>=3.4,<4
```

```suggestion
    filelock>=3.2,<4
```

Leave it as it is. `filelock >= 3.4` requires Python >= 3.6:

https://github.com/tox-dev/py-filelock/blob/a6c8fabc4192fa7a4ae19b1875ee842ec5eb4f61/setup.cfg#L34

but `virtualenv` supports:

https://github.com/pypa/virtualenv/blob/623189a00aa96c805990fd12dc883c69e251f4b2/setup.cfg#L21-L29

---

Thanks!

---

Indeed, but then to solve this we would want to have differing version requirements for 3.6+ and earlier; furthermore the code needs to dynamically generate the kwargs, otherwise on older pythons we'd be using kwarg that's not supported.